satpy/tests/reader_tests/test_mirs.py

```diff
@@ -259,24 +261,41 @@ def _check_attrs(data_arr, platform_name):
         [
             ([AWIPS_FILE], TEST_VARS, "metop-a"),
             ([NPP_MIRS_L2_SWATH], TEST_VARS, "npp"),
+            ([N20_MIRS_L2_SWATH], TEST_VARS, "noaa-20"),
+            ([OTHER_MIRS_L2_SWATH], TEST_VARS, "gpm"),
+
+            ([AWIPS_FILE], TEST_VARS, "metop-a"),
+            ([NPP_MIRS_L2_SWATH], TEST_VARS, "npp"),
+            ([N20_MIRS_L2_SWATH], TEST_VARS, "noaa-20"),
             ([OTHER_MIRS_L2_SWATH], TEST_VARS, "gpm"),
         ]
     )
-    def test_basic_load(self, filenames, loadable_ids, platform_name):
+    @pytest.mark.parametrize('reader_kw', [{}, {'limb_correction': False}])
+    def test_basic_load(self, filenames, loadable_ids,
+                        platform_name, reader_kw):
         """Test that variables are loaded properly."""
         from satpy.readers import load_reader
         with mock.patch('satpy.readers.mirs.xr.open_dataset') as od:
             od.side_effect = fake_open_dataset
             r = load_reader(self.reader_configs)
             loadables = r.select_files_from_pathnames(filenames)
-            r.create_filehandlers(loadables)
+            r.create_filehandlers(loadables,  fh_kwargs=reader_kw)
             with mock.patch('satpy.readers.mirs.read_atms_coeff_to_string') as \
                     fd, mock.patch('satpy.readers.mirs.retrieve'):
                 fd.side_effect = fake_coeff_from_fn
                 loaded_data_arrs = r.load(loadable_ids)
             assert loaded_data_arrs
+
             for data_id, data_arr in loaded_data_arrs.items():
+                sensor = data_arr.attrs['sensor']
+
                 if data_id['name'] not in ['latitude', 'longitude']:
                     self._check_area(data_arr)
                 self._check_fill(data_arr)
                 self._check_attrs(data_arr, platform_name)
+
+                sensor = data_arr.attrs['sensor']
```

Comment:
This is already set above (since your second to last commit). I think one of them can be removed.